litestar/contrib/minijinja.py

```diff
@@ -174,6 +174,32 @@ def register_template_callable(
         """
         self.engine.add_global(key, pass_state(template_callable))
 
+    def render_string(self, template_string: str, context: Mapping[str, Any] | None = None) -> str:
```

Comment:
Does the type for `context` need to be an optional mapping? From what I could tell, it's passed from the context we get from `create_template_context` which will always return a dictionary.

Reply:
I'm not smart enough to know the answer here 😅